docs/source/faq.rst

```diff
@@ -184,6 +184,16 @@ those on Hadoop, HPC, Kubernetes, and Cloud clusters.
 
     For more information see :doc:`deploying-kubernetes`
 
+4. **Managed cluster solutions**: These commercial options often handle creation and management of
```

Comment:
I'm not familiar with which of these (besides Coiled) actually support deploying on your own resources (your own cloud account / k8s). We should probably check on that.

Reply:
Ah good point. I changed this to simply `**Commercial Dask deployment:**`